Only in Vship-3.0.1: FFVship
diff -ru Vship-3.0.1-orig/Makefile Vship-3.0.1/Makefile
--- Vship-3.0.1-orig/Makefile	2025-08-12 11:10:20.487062125 -0700
+++ Vship-3.0.1/Makefile	2025-08-12 11:22:15.823497634 -0700
@@ -28,7 +28,7 @@
 	nvcc -x cu src/ffmpegmain.cpp -std=c++17  -arch=native $(subst -pthread,-Xcompiler="-pthread",$(ffvshiplibheader)) -o FFVship
 
 buildFFVSHIPall: src/ffmpegmain.cpp .FORCE
-	hipcc src/ffmpegmain.cpp -std=c++17 --offload-arch=gfx1100,gfx1101,gfx1102,gfx1103,gfx1030,gfx1031,gfx1032,gfx906,gfx801,gfx802,gfx803 -Wno-unused-result -Wno-ignored-attributes $(ffvshiplibheader) -o FFVship
+	hipcc src/ffmpegmain.cpp -std=c++17 --offload-arch=gfx1100,gfx1101,gfx1102,gfx1103,gfx1030,gfx1031,gfx1032,gfx906,gfx801,gfx802,gfx803 -Wno-unused-result -Wno-ignored-attributes $(ffvshiplibheader) $(shell pkg-config --cflags libavutil) -o FFVship
 
 buildFFVSHIPcudaall: src/ffmpegmain.cpp .FORCE
 	nvcc -x cu src/ffmpegmain.cpp -std=c++17 -arch=all $(subst -pthread,-Xcompiler="-pthread",$(ffvshiplibheader)) -o FFVship
