diff -ru neo_DFTTest-ec0d9473fc6652a22068966ddfef097253dd0256-orig/CMakeLists.txt neo_DFTTest-ec0d9473fc6652a22068966ddfef097253dd0256/CMakeLists.txt
--- neo_DFTTest-ec0d9473fc6652a22068966ddfef097253dd0256-orig/CMakeLists.txt	2025-12-04 07:47:36.896771947 -0800
+++ neo_DFTTest-ec0d9473fc6652a22068966ddfef097253dd0256/CMakeLists.txt	2025-12-04 07:47:53.900600115 -0800
@@ -17,10 +17,7 @@
 option(ENABLE_PAR "Enable C++17 Parallel Execution" ON)
 add_compile_definitions(VS_TARGET_CPU_X86 _CRT_SECURE_NO_WARNINGS _CRT_NONSTDC_NO_DEPRECATE)
 
-find_package(Git REQUIRED)
-execute_process(COMMAND ${GIT_EXECUTABLE} describe --first-parent --tags --always OUTPUT_VARIABLE GIT_REPO_VERSION)
-string(REGEX REPLACE "(r[0-9]+).*" "\\1" VERSION ${GIT_REPO_VERSION})
-execute_process(COMMAND ${GIT_EXECUTABLE} rev-list --count HEAD OUTPUT_VARIABLE GIT_COMMIT_COUNT OUTPUT_STRIP_TRAILING_WHITESPACE)
+set(VERSION "unknown")
 
 configure_file (
   "${PROJECT_SOURCE_DIR}/src/version.hpp.in"
